pxtextension.ts

```diff
@@ -107,12 +122,16 @@ namespace ml {
     event.onStopDetailedHandler = body;
   }
 
+  /**
+   * Tests if an ML event is currently detected.
+   * @param event one of the actions the machine learning model was trained on
+   */
```

Comment:
Should this have @return docs?

Reply:
Rob to review this and the other one to find out what's conventional and if it seems optional decide whether the text would add value to the existing text.

Reply:
`@returns` or `@return` do not appear to be a supported parameter for blocks. It appears in the tooltip for the block whereas `@param` does not. There are no other blocks examples I've found that state a return value in their docstrings. So we should probably leave it.
![image](https://github.com/user-attachments/assets/d96e5c82-06b0-4891-88a4-0bd3beb6a965)